documentation/jetty-documentation/src/main/asciidoc/operations-guide/server/server-logging-server.adoc

```diff
@@ -173,28 +165,34 @@ $JETTY_BASE
     └── logging-log4j2.ini
 ----
 
-The Jetty module system downloaded the required Log4j2 `+*.jar+` files, and created a `$JETTY_BASE/resources/log4j2.xml` file that you can configure to customize your Log4j2 logging.
+Jetty downloaded the required Log4j2 `+*.jar+` files, and created a `$JETTY_BASE/resources/log4j2.xml` file that you can configure to customize your Log4j2 logging.
+
+Please refer to the link:https://logging.apache.org/log4j/2.x/manual/configuration.html[Log4j2 configuration manual] for more information about how to configure Log4j2.
 
 [[og-server-logging-server-bridges]]
 ====== Bridging Logging to SLF4J
 
 When you use libraries that provide the features you need (for example, JDBC drivers), it may be possible that those libraries use a different logging framework than SLF4J.
 
 SLF4J provides link:http://www.slf4j.org/legacy.html[bridges for legacy logging APIs] that allows you to bridge logging from one of these legacy logging frameworks to SLF4J.
-Once the logging is bridged to SLF4J, you can use the xref:og-server-logging-server-default[default configuration] or the xref:og-server-logging-server-custom[custom configuration] so that your logging is centralized in one place only.
+Once the logging is bridged to SLF4J, you can use Jetty's xref:og-server-logging-server-default[default configuration] or a xref:og-server-logging-server-custom[custom configuration] so that your logging is centralized in one place.
+
+Jetty provides several modules out of the box for bridging older logging frameworks to SLF4J:
 
-Jetty provides out-of-the-box modules that you can enable to bridge logging from other logging frameworks to SLF4J.
+* `logging-jul-capture` for bridging from `java.util.logging`
```

Comment:
I was able to find these three modules by simply running `java -jar $JETTY_HOME/start.jar --list-modules=* | grep bridge`. Are there any others?

It looks like only `logging-jul-capture` is documented in the existing docs. If the other bridges are sufficiently similar in functionality, we may not have to fully document them separately, but it would probably be good to acknowledge them in a little more detail.

Reply:
I would rather mention that they are similar but very obsolete and should not be used anymore, and only document `jul` (which is in the JDK and will never go away).

Reply:
Got it. I added a warning to not use these older modules.